spec/Expressions.tex

```diff
@@ -298,6 +295,14 @@ \section{Precedence and Associativity}
 \hline
 \verb@new@ & right & initializer call \\
 \hline
+\verb@owned@ & right & apply management strategy to a class \\
+\verb@shared@ & right & \\
+\verb@borrowed@ & right & \\
+\verb@unmanaged@ & right & \\
```

Comment:
Let's leave just one `right` -- at `shared` or `owned` -- and remove it for the other management strategies.